tests/main.test.js

```diff
@@ -49,6 +48,19 @@ test('handles JS errors', async () => {
   }
 })
 
+test('handles impatient CSS downloads', async () => {
+  expect.assertions(1)
+  try {
+    await runMinimalcss('slowcss', {
+      // The most impatient option. The `slowcss.html` fixture uses
+      // a <link> href that takes a whole second to download.
+      waitUntil: 'load'
```

Comment:
It will generate errors most of the time. Maybe forbid this option?

Reply:
Good point. You're right, it'd happen even on a page that doesn't have any slow XHR requests. Hmm... Can there be any value in ever running with `'load'`?

Reply:
> Can there be any value in ever running with 'load'?

I can't see any